RealmModelGenerator/DetailsMainVC.swift

```diff
@@ -8,36 +8,16 @@
 
 import Cocoa
 
-protocol DetailsVCDelegate: class {
-    func detailsVC(detailsVC:DetailsMainVC, detailObject:AnyObject, detailType:DetailType)
-}
-
-class DetailsMainVC: NSViewController, EntityDetailVCDelegate, AttributeDetailVCDelegate, RelationshipDetailVCDelegate {
+class DetailsMainVC: NSViewController {
     static let TAG = NSStringFromClass(DetailsMainVC)
     
-    var entityDetailVC: EntityDetailVC! = nil {
-        didSet {
-            self.entityDetailVC.delegate = self
-        }
-    }
-    
-    var attributeDetailVC: AttributeDetailVC! = nil {
-        didSet {
-            self.attributeDetailVC.delegate = self
-        }
-    }
-    
-    var relationshipDetailVC: RelationshipDetailVC! = nil {
-        didSet {
-            self.relationshipDetailVC.delegate = self
-        }
-    }
+    private var entityDetailVC: EntityDetailVC! = nil
+    private var attributeDetailVC: AttributeDetailVC! = nil
+    private var relationshipDetailVC: RelationshipDetailVC! = nil
     
     @IBOutlet weak var detailsContainerView: NSView!
     @IBOutlet weak var emptyTextField: NSTextField!
     
-    weak var delegate: DetailsVCDelegate?
-    
```

Comment:
self.storyboard?.instantiateController should be self.storyboard!. instantiateController. 
if the vc doesn't have a storyboard then something bad happened